runtime/gc_vlhgc/MemorySubSpaceTarok.hpp

```diff
@@ -299,6 +299,8 @@ class MM_MemorySubSpaceTarok : public MM_MemorySubSpace
 	 */
 	bool consumeFromTaxationThreshold(MM_EnvironmentBase *env, uintptr_t bytesToConsume);
 
+	uintptr_t releaseFreeMemoryPages(MM_EnvironmentBase* env, uintptr_t memoryType);
```

Comment:
I'd add an explicit virtual here (although effectively it already is, since it's declared so in base)